implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -79,6 +79,7 @@ private static JWTAuthContextInfoProvider create(String publicKey, String public
         provider.mpJwtRequireIss = Optional.of(Boolean.TRUE);
         provider.tokenHeader = AUTHORIZATION_HEADER;
         provider.tokenCookie = Optional.empty();
+        provider.alwaysCheckAuthorization = false;
```

Comment:
@nicolaimainiero Can you drop this one please, I know I've asked you to do it but I forgot the default value was 'false', my apologies

Reply:
Sure. I was unsure if it was necessary because some properties are configured as default values and some in this method. Nevertheless, I have now removed it.